website/project/metadata/erpc-prize.json

```diff
@@ -0,0 +1,151 @@
+            "required": true
+        }, {
+            "qid": "q3",
+            "nav": "Research Questions",
+            "type": "string",
+            "format": "textarea",
+            "title": "Research Questions",
+            "description": "Please list each research question included in this study.",
+            "required": true
+        }, {
+            "qid": "q4",
+            "nav": "Hypotheses",
+            "type": "string",
+            "format": "textarea",
+            "title": "Hypotheses",
+            "description": "For each of the research questions listed in the previous section, provide one or multiple specific and testable hypotheses. Please state if the hypotheses are directional or non-directional. If directional, state the direction. A predicted effect is also appropriate here.",
+            "required": true
+        }]
+    }, {
+        "id": "page2",
+        "title": "Variables",
+        "description": "In this section you can describe all variables (both manipulated and measured variables) that will later be used in your confirmatory analysis plan. In your analysis plan, you will have the opportunity to describe how each variable will be used. If you have variables which you are measuring for exploratory analyses, you are not required to list them, though you are permitted to do so.",
+        "questions": [{
+            "qid": "q5",
+            "nav": "Measured variables",
+            "title": "Measured variables",
+            "description": "Describe each variable that you will measure. This will include outcome measures, as well as any predictors or covariates that you will measure.",
+            "type": "string",
+            "format": "textarea",
+            "required": true
+        }, {
+            "qid": "q6",
+            "nav": "Indices",
+            "type": "string",
+            "format": "textarea",
+            "title": "Indices",
+            "description": "If any measurements are  going to be combined into an index (or even a mean), what measures will you use and how will they be combined? Include either a formula or a precise description of your method. If your are using a more complicated statistical method to combine measures (e.g. a factor analysis), you can note that here but describe the exact method in the analysis plan section.",
+            "required": false
+        }]
+    }, {
+        "id": "page3",
+        "title": "Analysis Plan",
+        "description": "<p>You may describe one or more analyses in this preregistration. Please remember that all analyses specified below must be reported in the final article, and any additional analyses must be noted as exploratory or hypothesis generating.</p><p> An analysis plan must state up front which variables are predictors (independent) and which are the outcomes (dependent), otherwise it is an exploratory analysis. You are allowed to describe any exploratory work here, but a clear confirmatory analysis is required.</p>",
```

Comment:
Typos: 

- "hypothesis-generating"
- "upfront"
- "...and which are the outcomes (dependent)**;** otherwise**,** it is an exploratory analysis."